### DISCUSSION TITLE
Track Package Manager selection on docs

### DISCUSSION THREAD

**Discussion by doganalper** [nuxt/nuxt#25785] [Ideas]
I think it would be more user friendly if on package manager selection, (pnpm, yarn etc.) it stores selection on localStorage and next time visiting, docs start with last package manager selection.
On another note, can it be synced with other `code-group` sections so on selection it automatically selects that package manager on others too.

**Comment by TheAlexLichter** [comment]
This should probably be used for https://github.com/nuxt/nuxt.com ☺️  -> https://github.com/nuxt/nuxt.com/issues/1501

**Reply by doganalper** [reply]
True! Sorry for misplacement 😅

**Reply by doganalper** [reply]
For some reason I cannot transfer this to that repository, [nuxt/nuxt.com](https://github.com/nuxt/nuxt.com) does not show up on `Transfer this discussion`

**Reply by TheAlexLichter** [reply]
For me neither. Just re-open there 👍

**Reply by TheAlexLichter** [reply]
https://github.com/nuxt/nuxt.com/issues/1501
